slicedimage/_tile.py

```diff
@@ -80,3 +125,13 @@ def copy(self, dst_fh):
             self.tile_format = ImageFormat.NUMPY
         else:
             raise RuntimeError("copy can only be called on a tile that hasn't been decoded.")
+
+    def validate(self):
+        """
+        Assert the calculated sha256 matches the one provided on object creation.
+        A missing expected sha256 value is considered non-fatal.
+        """
+        self._load()
```

Comment:
I think _load will now validate against incorrect checksums.

Reply:
That's definitely what I was seeing in https://github.com/spacetx/slicedimage/pull/61#issuecomment-420242372